hawkbit-runtime/hawkbit-update-server/pom.xml

```diff
@@ -59,6 +59,16 @@
          <groupId>com.microsoft.sqlserver</groupId>
          <artifactId>mssql-jdbc</artifactId>
       </dependency>
+      <!-- TEST -->
+      <dependency>
+         <groupId>org.springframework.boot</groupId>
+         <artifactId>spring-boot-starter-test</artifactId>
```

Comment:
I think there is a scope `test` missing but I'd rather prefer to change `hawkbit-parent`'s pom.xml

Reply:
Yes, thank you

Reply:
Welcome. May I ask you to add the `test` scope [to the managed location](https://github.com/eclipse/hawkbit/blob/master/pom.xml#L756) as well?
I think it would be good if `test` is the default scope since it is a "test-only" dependency